src/components/CreateOrModifyTagDialog.tsx

```diff
@@ -62,7 +62,7 @@ export default function CreateOrModifyTagDialog({
 			description: formValues.description || undefined,
 		};
 
-		currentTagId === 'new'
+		currentTagId === null
```

Comment:
Are you sure that the falsy value of currentTagId will only have a null value? What about undefined?

Reply:
Yes. But first i have to change pass value condition for currentTagId. First I have to check if the modalProps is not undefined then i pass a string value. When not I pass null